StripeConnect/StripeConnect.xcodeproj/project.pbxproj

```diff
@@ -276,6 +302,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				4161C2792C9DB1CE005BD67C /* StripeUICore.framework in Frameworks */,
```

Comment:
🚫 **Blocker**
We also need to add the `StripeUICore` dependency to the Example app or Xcode Cloud will fail to build